README.md

```diff
@@ -130,39 +148,38 @@ curl 'http://127.0.0.1:7700/indexes/movies/search?q=botman+robin&limit=2' | jq
 }
 ```
 
-#### With the Web Interface
+#### Use the Web Interface
 
-MeiliSearch provides a simple web interface containing a search bar in order to quickly test the instant search experience with a given set of documents.
+We also deliver an **out-of-the-box web interface** in which you can test MeiliSearch interactively.
 
-This web interface is available in your browser at the root of the server. The default URL is [http://127.0.0.1:7700](http://127.0.0.1:7700).
+You can access the web interface in your web browser at the root of the server. The default URL is [http://127.0.0.1:7700](http://127.0.0.1:7700). All you need to do is open your web browser and enter MeiliSearch’s address to visit it. This will lead you to a web page with a search bar that will allow you to search in the selected index.
```

Comment:
Hey @bidoubiwa!

This would be great if we can find a screenshot or a gif to illustrate the web UI.

Reply:
There is one! https://docs.meilisearch.com/guides/advanced_guides/web_interface.html. Maybe it is a little outdated. Would you prefer a new one?

Reply:
No that seems great, could you do a github suggestion to add the gif, please ?

Reply:
```suggestion
You can access the web interface in your web browser at the root of the server. The default URL is [http://127.0.0.1:7700](http://127.0.0.1:7700). All you need to do is open your web browser and enter MeiliSearch’s address to visit it. This will lead you to a web page with a search bar that will allow you to search in the selected index.

![web interface gif](https://docs.meilisearch.com/movies-web-demo.gif)
```